### DISCUSSION TITLE
How to open location in Organic Maps?

### DISCUSSION THREAD

**Discussion by retiolus** [organicmaps/organicmaps#8122] [Q&A] [answered]
Hi, I'm building an app ([osm2gmaps](https://codeberg.org/retiolus/osm2gmaps)) and I need to open a location in Organic Maps app, but when I [try to do that](https://codeberg.org/retiolus/osm2gmaps/src/commit/9c74e2948371fc790861998b248ca37b05f8cf40/app/src/main/java/net/retiolus/osm2gmaps/activities/MainActivity.kt#L375) I get "The file type is not recognized by the app".

I first tried with `om://` link and now with `https://omaps.app/` but I get the same: https://codeberg.org/retiolus/osm2gmaps/src/commit/9c74e2948371fc790861998b248ca37b05f8cf40/app/src/main/java/net/retiolus/osm2gmaps/utils/maps/converters/OrganicMapsConverter.kt#L126

**Comment by biodranik** [comment, marked as answer]
This format is not yet supported in OM clients, any volunteering help is welcomed! See https://github.com/organicmaps/organicmaps/issues/7482

Check other ways to open it here: https://omaps.app/api and here https://github.com/organicmaps/api-android/

**Reply by retiolus** [reply]
Thanks for your answer! So basically, I have to encode the URL before so it can be opened with Organic Maps!

**Reply by retiolus** [reply]
By the way, where can I find the encoding a decoding algorithms?

**Reply by biodranik** [reply]
Here: https://github.com/organicmaps/organicmaps/tree/master/ge0
